app/routes.py

```diff
@@ -113,11 +110,13 @@ def create_shortlink():
         domain,
         base_path
     )
-    base_response_url = check_params(scheme, domain, url, base_path)
+    check_params(url)
```

Comment:
The variables above `scheme`, `domain` and `base_path` are not needed anymore and should be removed.